easybuild/easyconfigs/p/pigz/pigz-2.4_Makefile.patch

```diff
@@ -0,0 +1,19 @@
+diff -Nru pigz-2.4.orig/Makefile pigz-2.4/Makefile
+--- pigz-2.4.orig/Makefile	2017-12-27 00:35:01.000000000 +0000
++++ pigz-2.4/Makefile	2018-08-13 17:35:26.000000000 +0100
+@@ -1,5 +1,4 @@
+-CC=gcc
+-CFLAGS=-O3 -Wall -Wextra -Wno-unknown-pragmas
++ZLIB=-L$(EBROOTZLIB)/lib
```

Comment:
Hmm, I don't see why this is necessary... Shouldn't the right `libz.*` get picked up via `$LIBRARY_PATH`, which specifies locations for the linker to consider?